### PR TITLE
Product Index - Create

### PR DESCRIPTION
**Pull request by phanlyhuynh**
Index, create product

### CODE REVIEW

**Comment by phanlyhuynh** [comment]
ready

File: app/Http/Controllers/Admin/ProductController.php [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+        {
+            $imgs = $request->file('images');
+            foreach ($imgs as $fileimg) {
+                $fName = md5(date('Y-m-d H:i:s') . rand(0, 10000)) . '.' . $fileimg->getClientOriginalExtension();
+                $fileimg->move(public_path('uploads/products/details'), $fName);
+                $img = new Image();
+                $img->link = $fName;
+                $product->images()->save($img);
+            }
+        }
```
Comment (BunnyPi04) [inline app/Http/Controllers/Admin/ProductController.php]: viết vào scope nhé

**Comment by BunnyPi04** [comment]
commented

**Comment by phanlyhuynh** [comment]
ready

File: resources/views/admin/product/create.blade.php [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        </div>
+    </div>
+    <div class="card">
+        <div class="card-header border bottom">
+            <h4 class="card-title">{{ trans('home.addnew') }}</h4>
+        </div>
+        <div class="card-body">
+            <div class="row">
+                <div class="col-sm-8">
+                    {!! Form::open(['method' => 'POST','url' => 'admin/products', 'files' => true, 'id' => 'form-validation', 'novalidate' => 'novalidate', 'role' => 'form']) !!}
```
Comment (BunnyPi04) [inline resources/views/admin/product/create.blade.php]: convention

File: resources/views/admin/product/index.blade.php [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+                                                @endif
+                                            </td>
+                                            <td class="text-center font-size-18">
+                                                {!! Form::open(['method' => 'GET', 'url' => 'admin/products/' . $product->id . '/edit']) !!}
+                                                <button class="text-gray btn btn-float">
+                                                    <i class="ti-pencil"></i>
+                                                </button>
+                                                {!! Form::close() !!}
+                                                {!! Form::open(['method' => 'DELETE', 'url' => ['admin/products', $product->id]]) !!}
+                                                <button type="submit" class="text-gray btn btn-float" onclick="return confirm('{{ trans('home.confirm_delete') }}');">
```
Comment (BunnyPi04) [inline resources/views/admin/product/index.blade.php]: sử dụng laravel collective

**Comment by BunnyPi04** [comment]
commented

**Comment by phanlyhuynh** [comment]
ready

**Pull request merged**